src/schema/objects/datatypes.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 # This file defines valid BIDS datatypes, along with their full names and descriptions.
-# For rules regarding how suffixes relate to datatypes, see files in `rules/datatypes/`.
+# For rules regarding how suffixes relate to datatypes, see files in `rules/files/raw`.
```

Comment:
Thanks for catching this!  I have recommended it now as a part of dedicated https://github.com/bids-standard/bids-specification/pull/1362